api.md

```diff
@@ -748,6 +748,42 @@ For a table with more than one space dimension:
 SELECT set_number_partitions('conditions', 2, 'device_id');
 ```
 
+
+--
+
+## set_integer_now_func() [](set_integer_now_func)
```

Comment:
This function needs to be added to the command list at the top of the file

Reply:
done